static/app/views/performance/table.tsx

```diff
@@ -118,7 +149,14 @@ class Table extends React.Component<Props, State> {
       return dataRow[column.key];
     }
     const tableMeta = tableData.meta;
-
+    // Attach sort to the metadata if sorting by a span operation. This is so fieldRenderer will know which breakdown to display first.
+    if (
+      location.query.sort &&
+      typeof location.query.sort === 'string' &&
+      SPAN_OP_BREAKDOWN_FIELDS.includes(location.query.sort.replace(/^-/, ''))
+    ) {
+      dataRow.sortedBy = location.query.sort.replace(/^-/, '');
```

Comment:
Can the sort be retrieved from `eventView`?

Reply:
This way of attaching the sort to the `dataRow` is really gross 😬. I wonder if it makes more sense to just pass the `eventView` to the `fieldRenderer`.

Reply:
👍  on retrieving from the `eventView`.

Also agree on the `fieldRenderer` stuff. I think I'll just add `eventView` as optional to `RenderFunctionBaggage`

Reply:
Moved out table logic into it's own file at `static/app/views/performance/transactionSummary/transactionEvents/eventsTable.tsx`!
Also refactored code to pass `eventView` to `fieldRenderer` via `RenderFunctionBaggage` rather than mutating `dataRow`.